app/src/lib/stores/helpers/pull-request-updater.ts

```diff
@@ -0,0 +1,124 @@
+import { PullRequestStore } from '../pull-request-store'
+import { Account } from '../../../models/account'
+import { fatalError } from '../../fatal-error'
+import { PullRequest } from '../../../models/pull-request'
+import { GitHubRepository } from '../../../models/github-repository'
+
+const PullRequestInterval = 1000 * 60 * 3
```

Comment:
Can we :memo: these consts to make it clear what each is used for, and why they differ?